src/vs/base/browser/ui/list/listWidget.ts

```diff
@@ -236,7 +236,7 @@ class KeyboardController<T> implements IDisposable {
 		private view: ListView<T>,
 		options: IListOptions<T>
 	) {
-		const multipleSelectionSupport = !(options.multipleSelectionSupport === false);
+		const multipleSelectionSupport = options.multipleSelectionSupport !== false;
```

Comment:
I will do the other programming changes requested, but this the one unrelated line is not only poor style, but the only one of its kind (every other statement in this entire program like this uses !==). I understand why the other refactorings should be in a different PR and undone but this sticks out like a sore thumb.

Only time I will ever say this.